client/reader/stream/post-unavailable.jsx

```diff
@@ -4,26 +4,20 @@ import { localize } from 'i18n-calypso';
 import { PureComponent } from 'react';
 
 class PostUnavailable extends PureComponent {
-	componentDidMount() {
-		this.errors = {
-			unauthorized: this.props.translate(
-				'This is a post on a private site that you’re following, but not currently a member of.' +
-					' ' +
-					'Please request membership to display these posts in Reader.'
-			),
-			default: this.props.translate( 'An error occurred loading this post.' ),
-		};
-	}
```

Comment:
I couldn't find a plausible way these error messages get shown now, so refactored for simplicity.